telco-hub/configuration/reference-crs/required/gitops/ztp-installation/argocd-openshift-gitops-patch.json

```diff
@@ -0,0 +1,92 @@
+{
+  "spec": {
+    "controller": {
+      "resources": {
+        "limits": {
+          "cpu": "16",
+          "memory": "32Gi"
+        },
+        "requests": {
+          "cpu": "1",
+          "memory": "2Gi"
+        }
+      }
+    },
+    "kustomizeBuildOptions": "--enable-alpha-plugins",
+    "repo": {
+      "volumes": [
+        {
+          "name": "kustomize",
+          "emptyDir": {}
+        }
+      ],
+      "initContainers": [
+        {
+          "resources": {
+          },
+          "terminationMessagePath": "/dev/termination-log",
+          "name": "kustomize-plugin",
+          "command": [
+            "/exportkustomize.sh"
+          ],
+          "args": [
+            "/.config"
+          ],
+          "imagePullPolicy": "Always",
+          "volumeMounts": [
+            {
+              "name": "kustomize",
+              "mountPath": "/.config"
+            }
+          ],
+          "terminationMessagePolicy": "File",
+          "image": "registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.16.2"
```

Comment:
Should be 4.17.0-25 until a 4.18 version is available.

Reply:
done after running the script with 4.17.